data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/transformer/OrganizationTransformers.java

```diff
@@ -29,7 +28,7 @@ public DataEnvelope processData(OrganizationSp organizationSp, OrganizationType
         processor.processOrgFax(organizationSp.getOrganizationFax(), transformedObj);
         processor.processOrgEntity(organizationSp.getOrganizationEntityId(), transformedObj);
         processor.processOrgName(organizationSp.getOrganizationName(), transformedObj);
-        return jsonGenerator.buildAvroRecord(transformedObj);
+        return jsonGenerator.generateStringJson(transformedObj);
```

Comment:
Make sure the generatedStringJson method in the commonUtils is updated to mark the key for organization, patient and provider data.

Reply:
I already have a condition to mark the key to be `optional: false`